src/styles/FooterStyles.scss

```diff
@@ -0,0 +1,45 @@
+$btn-color-bg: #0073c8;
+$btn-color-bg--hover: #fff;
+
+.footer-section {
+  padding-top: 4rem;
+
+  hr {
+    width: 100vw;
+    position: relative;
+    left: calc(-50vw + 50%);
+  }
+
+  footer {
+    .global-footer-container {
+      .btn {
+        &.btn.btn-subscribe {
+          border-width: 1px;
+          background-color: $btn-color-bg;
+
+          &:hover {
+            background-color: $btn-color-bg--hover;
+          }
+        }
+      }
+
+      .social-container {
+        svg {
+          display: inline;
+        }
+      }
+
+      .footer-social-newsletter .subscriber-form .form-control {
+        height: 42px;
+      }
+    }
+
+    .qpp-hhs-logo-container .qpp-logo-container .madie-logo {
+      width: 280px;
+    }
+
+    .qpp-hhs-logo-container {
+      overflow: hidden;
+    }
```

Comment:
this is probably now not being used, right?

Reply:
It is adjusting a component from madie-design-system so it matched the design comp.

Reply:
ah..okay, thanks for the clarification!